CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 3.6.O - [#66](https://github.com/openfisca/country-template/pull/66)
```

Comment:
Isn't it a breaking change? (I'll approve if we agree it is not, or if we bump major)

Reply:
Which usecase are you thinking about?

Reply:
I guess we could consider that a BC if we consider that users can run tests of their own that are not part of the package 🤔

Would you have time to suggest or add a few commits to explicit the breaking change?

Reply:
Hey, after thinking a bit in the metro, and by the way the library is used, I guess you're right is a minor.